cdap-common/src/main/java/io/cdap/cdap/internal/app/store/RunRecordMeta.java

```diff
@@ -37,6 +37,7 @@
  * includes information that should not be exposed to users. {@link RunRecord} contains fields that are exposed
  * to users, so everything else like the Twill runid should go here.
  */
+// TODO: rename RunRecordMeta to RunRecordDetail
```

Comment:
Why?

Reply:
In the past, normally I see "metadata" terminology is used to refer to data that provides information about other data and it typically doesn't encapsulate the data it provides information for. But RunRecordMetada here extends and includes the RunRecord. So hope to use xxxDetail to name these classes that encapsulate both the data and metadata.

Reply:
Generally I would prefer as few TODO as possible in the code (you either do it now, or not to do it). I have no objection in renaming the class, but we don't need to add a TODO for it.

Reply:
Sounds good. Done. Renamed RunRecordMeta -> RunRecordDetail. 

Was hesitated as it affects quite a lot of files which makes the PR difficult to review.